### PR TITLE
Import with new i18n fields: 'labels' and 'names'

### PR DESCRIPTION
**Pull request by jbgriesner**
As title says this is linked with the i18n needs in mimir.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,11 +6,11 @@ edition = "2018"
 
 [dependencies]
 log = "0.4"
-geo = "0.6"
+geo-types = "0.4"
 postgres = "0.15"
 fallible-iterator = "0.1"
-mimirsbrunn = { git = "https://github.com/CanalTP/mimirsbrunn.git", rev = "v1.7.0" }
-mimir = { git = "https://github.com/CanalTP/mimirsbrunn.git", rev = "v1.7.0" }
+mimirsbrunn = { git = "https://github.com/jbgriesner/mimirsbrunn", branch = "i18n_import_poi" }
```
Comment (jbgriesner) [inline Cargo.toml]: TODO: changes with the 'official' branch once merged.

File: Cargo.toml [hunk 2]
```diff
@@ -6,11 +6,11 @@ edition = "2018"
...
+mimir = { git = "https://github.com/jbgriesner/mimirsbrunn", branch = "i18n_import_poi" }
```
Comment (jbgriesner) [inline Cargo.toml]: TODO: changes with the 'official' branch once merged.

File: Cargo.toml [hunk 3]
```diff
@@ -21,8 +21,8 @@ num_cpus = "1.8.0"
 [dev-dependencies]
 retry = "*"
 hyper = "0.10"
-bragi = { git = "https://github.com/CanalTP/mimirsbrunn.git", rev = "v1.7.0" }
-cosmogony = "0.3"
+bragi = { git = "https://github.com/jbgriesner/mimirsbrunn", branch = "i18n_import_poi" }
```
Comment (jbgriesner) [inline Cargo.toml]: TODO: changes with the 'official' branch once merged.

File: tests/fafnir_tests/mod.rs [hunk 4]
```diff
@@ -431,7 +434,7 @@ pub fn main_test(mut es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapp
 
     // Test that two "Tour Eiffel" POI should have been imported: the hotel + the monument
     let eiffels: Vec<mimir::Place> = es_wrapper
-        .search_and_filter("Tour Eiffel", |_| true)
+        .search_and_filter("name:Tour Eiffel", |_| true)
```
Comment (amatissart) [inline tests/fafnir_tests/mod.rs]: In `name:Tour Eiffel` the "name" field only applies to "Tour" and not "Eiffel"

File: src/lib.rs [hunk 5]
```diff
@@ -45,6 +44,33 @@ fn build_poi_properties(row: &Row, name: &str) -> Result<Vec<Property>, String>
...
+    const NAME_TAG_PREFIX: &str = "name:";
+
+    let properties = properties
+        .iter()
+        .filter(|property| property.key.starts_with(&NAME_TAG_PREFIX))
+        .map(|property| mimir::Property {
+            key: property.key[NAME_TAG_PREFIX.len()..].to_string(),
+            value: property.value.to_string(),
+        })
+        .filter(|p| langs.contains(&p.key))
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply src/lib.rs]: Or directly use `filter_map` (would be better).

File: src/lib.rs [hunk 6]
```diff
@@ -375,19 +424,20 @@ pub fn load_and_index_pois(
...
-            build_poi(p)
+            build_poi(p, &langs)
                 .ok_or_else(|| warn!("Problem occurred in build_poi()"))
                 .ok()
         })
         .pack(1000)
         .with_nb_threads(nb_threads)
         .par_map({
             let i = poi_index.clone();
+            let langs = langs.clone();
```
[the remark this answers is not in the document]
Comment (jbgriesner) [reply src/lib.rs]: cuz inside a 'par_map' each thread will have its own 'langs' variable

[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply src/lib.rs]: You're not updating it, therefore I don't understand why you don't just use it? You're already in the thread in here after all...

**Pull request merged**